base/setup/usetup/lang/es-ES.h

```diff
@@ -1,7 +1,7 @@
 // This file is converted by code7bit.
 // code7bit: https://github.com/katahiromz/code7bit
 // To revert conversion, please execute "code7bit -r <file>".
```

Comment:
Did you de-convert this file before translation and then convert it back via the code7bit tool? It's important, because without doing that, by simple editing a file, the changed text will not be displayed correctly. Personally, when I changed usetup translations, they was displayed correctly only after I did the conversion. So asking just for sure.

Reply:
This part of the file doesn't have changes. But I remember that I converted it correctly.

Reply:
OK then. Thanks.